js/project-card.js

```diff
@@ -1,16 +1,9 @@
 /* Project Card */
 
 var ProjectCard = {
-  ID_PREFIX: "p",
   template: _.template($("script#project-card-template").html()),
-  getProjectCardId: function(projectData) {
```

Comment:
I moved thi method to `PulseMaker` in `pulse.js`. I think `id` should be part of the project `object`... not just as part of the project card template data.